src/index.js

```diff
@@ -27,6 +27,20 @@ const COLOR_SCALES = Object.freeze({
 });
 /* eslint-enable max-len */
 
+function setDimensionLimit(handler) {
+  var dims = handler.getDimensions();
+  if(dims && dims.length) {
+    dims.forEach(function(d, i){
+      if(i === 1) {
+        d.qOtherTotalSpec.qOtherMode = 'OTHER_COUNTED';
+        d.qOtherTotalSpec.qOtherCounted = { qv: '20' };
+      } else {
+        delete d.qOtherTotalSpec;
+      }
+    });
+  }
+}
+
```

Comment:
I think the intention of this function is unclear. What is special about i == 1? The naming is also unusual, because the function is not a setter. Something like updateDimensionLimit would be more appropriate.